src/applications/coronavirus-screener/components/FormQuestion.jsx

```diff
@@ -1,56 +1,32 @@
 import React from 'react';
-import recordEvent from 'platform/monitoring/record-event';
 import _ from 'lodash/fp';
 import classnames from 'classnames';
-import moment from 'moment';
 
 export default function FormQuestion({
   question,
-  formState,
-  setFormState,
-  resultSubmitted,
-  setResultSubmittedState,
   scrollNext,
+  recordStart,
+  optionsConfig,
+  setQuestionValue,
 }) {
-  function handleChange(event) {
-    if (_.isEmpty(formState)) {
-      recordEvent({
-        event: 'covid-screening-tool-start',
-        'screening-tool-question': question.id,
-      });
-      // starts duration timer for GA
-      setResultSubmittedState({
-        ...resultSubmitted,
-        startTime: moment().unix(),
-      });
-    }
-    if (question.id === 'isStaff' && !_.isEmpty(formState)) {
-      setFormState({ [question.id]: event.target.value });
-    } else {
-      // sets the current question value in form state
-      setFormState({
-        ...formState,
-        [question.id]: event.target.value,
-      });
-    }
+  function handleClick(event) {
+    recordStart(question.id);
+    setQuestionValue({ event, questionId: question.id });
     scrollNext();
   }
 
-  const optionsConfig = [
-    { optionValue: 'yes', optionText: 'Yes' },
-    { optionValue: 'no', optionText: 'No' },
-  ];
+  // console.log('question', question);
+
```

Comment:
Does the linter catch debug statements? 🤔

Reply:
yup it does, i need to set up my IDE for debugging/breakpoints instead of using these